### DISCUSSION TITLE
Welcome to Merklin Discussions!

### DISCUSSION THREAD

**Discussion by toniengelhardt** [toniengelhardt/merklin#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RyanPaulGannon** [comment]
Yo, I've been searching for Vue/Nuxt projects to contribute too as well as pretty much anything web3. 😅

I've been attempting to freelance for a year (learn Js/Ts since early '21) and currently just looking for interesting projects to contribute too. Came across this, and I'd like to help!

**Reply by toniengelhardt** [reply]
Hi @RyanPaulGannon, I was there a week ago, that's why I started this project lol. 

It's still very basic and I'm not sure how much time I'll have to work on it. But feel free to fork it, play around, open issues, and PRs. The idea is to build it as modular as possible with the tiles (a bit like Dune), so that it is easy collaborate.

Btw I also started compiling this list: [web3-vue-and-nuxt-projects](https://github.com/toniengelhardt/web3-vue-and-nuxt-projects)

It's time to bring Vue/Nuxt to the web3 stack ✌🏽

**Reply by RyanPaulGannon** [reply]
Yeah man, agreed, more Vue/Nuxt in web3 🙌🏻

The app looks cool mate, looking forward to having a play around!

**Reply by toniengelhardt** [reply]
Let's go!
